.github/workflows/test.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Install dependencies and package
         run: |
           CUDA_TAG=cpu pip install -r requirements.torch.txt --find-links https://download.pytorch.org/whl/torch_stable.html
-          pip install -r requirements.dev.txt
+          pip install .[dev]
           pip install cython
           pip install -e .
```

Comment:
Can't we replace `pip install .[dev]` and `pip install -e .` by `pip install -e .[dev]`?

Reply:
And if `pip install .[dev]` works without error, we might no longer need `pip install cython`.